common/annotations/en.xml

```diff
@@ -2942,77 +2942,77 @@ annotations.
 		<annotation cp="🛍" type="tts">shopping bags</annotation>
 		<annotation cp="🎒">backpack | backpacking | bag | bookbag | education | rucksack | satchel | school</annotation>
 		<annotation cp="🎒" type="tts">backpack</annotation>
-		<annotation cp="🩴">beach | flip flop | sandal | sandals | shoe | thong sandal | thong sandals | thongs | zōri</annotation>
+		<annotation cp="🩴">beach | flip | flop | sandal | sandals | shoe | thong | thongs | zōri</annotation>
 		<annotation cp="🩴" type="tts">thong sandal</annotation>
-		<annotation cp="👞">brown | clothes | clothing | feet | foot | kick | man | man’s shoe | shoe | shoes | shopping</annotation>
+		<annotation cp="👞">brown | clothes | clothing | feet | foot | kick | man | man’s | shoe | shoes | shopping</annotation>
 		<annotation cp="👞" type="tts">man’s shoe</annotation>
 		<annotation cp="👟">athletic | clothes | clothing | fast | kick | running | shoe | shoes | shopping | sneaker | tennis</annotation>
 		<annotation cp="👟" type="tts">running shoe</annotation>
-		<annotation cp="🥾">backpacking | boot | brown shoe | camping | hiking | outdoors | shoe</annotation>
+		<annotation cp="🥾">backpacking | boot | brown | camping | hiking | outdoors | shoe</annotation>
 		<annotation cp="🥾" type="tts">hiking boot</annotation>
-		<annotation cp="🥿">ballet flat | comfy | flat shoe | flats | slip-on | slipper</annotation>
+		<annotation cp="🥿">ballet | comfy | flat | flats | shoe | slip-on | slipper</annotation>
 		<annotation cp="🥿" type="tts">flat shoe</annotation>
-		<annotation cp="👠">clothes | clothing | dress | fashion | heel | heels | high-heeled shoe | shoe | shoes | shopping | stilletto | woman</annotation>
+		<annotation cp="👠">clothes | clothing | dress | fashion | heel | heels | high-heeled | shoe | shoes | shopping | stilletto | woman</annotation>
 		<annotation cp="👠" type="tts">high-heeled shoe</annotation>
-		<annotation cp="👡">clothing | sandal | shoe | woman | woman’s sandal</annotation>
+		<annotation cp="👡">clothing | sandal | shoe | woman | woman’s</annotation>
 		<annotation cp="👡" type="tts">woman’s sandal</annotation>
-		<annotation cp="🩰">ballet | ballet shoes | dance</annotation>
+		<annotation cp="🩰">ballet | dance | shoes</annotation>
 		<annotation cp="🩰" type="tts">ballet shoes</annotation>
-		<annotation cp="👢">boot | clothes | clothing | dress | shoe | shoes | shopping | woman | woman’s boot</annotation>
+		<annotation cp="👢">boot | clothes | clothing | dress | shoe | shoes | shopping | woman | woman’s</annotation>
 		<annotation cp="👢" type="tts">woman’s boot</annotation>
 		<annotation cp="🪮">Afro | comb | groom | hair | pick</annotation> <!-- 1FAAE -->
 		<annotation cp="🪮" type="tts">hair pick</annotation>
-		<annotation cp="👑">clothing | crown | king | medieval | queen | royal | royal family | royalty | win</annotation>
+		<annotation cp="👑">clothing | crown | family | king | medieval | queen | royal | royalty | win</annotation>
```

Comment:
Family by itself seems odd in context of a crown.

Reply:
Fixed in spreadsheet: came from 'royal family'